### PR TITLE
Added missing sideloading script to Ts and Js projects.

### PR DESCRIPTION
**Pull request by MsCesarBerard**

### CODE REVIEW

**Comment by MsCesarBerard** [comment]
@akrantz is added to the review. #Closed

**Comment by MsCesarBerard** [comment]
@TCourtneyOwen is added to the review. #Closed

File: src/app/templates/js/angular/package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "version": "0.1.0",
   "scripts": {
     "start": "webpack-dev-server --mode development --https --key ./certs/server.key --cert ./certs/server.crt --cacert ./certs/ca.crt --port 3000",
+    "sideload": "node node_modules/office-toolbox/app/office-toolbox.js sideload -m <%= projectInternalName %>-manifest.xml -a <%= hostInternalName %>",
```
Comment (akrantz) [inline src/app/templates/js/angular/package.json]: This can be simplified to "sideload": "office-toolbox sideload -m …" (in all of the package.json files)

**Comment by akrantz** [comment]
@MsCesarBerard Did the latest office-toolbox (v0.1.0) sideload command work ok?

**Comment by MsCesarBerard** [comment]
Yes, I have tried several types of projects on different hosts and it works.

---
In reply to: [402313304](https://github.com/OfficeDev/generator-office/pull/335#issuecomment-402313304) [](ancestors = 402313304)

**Comment by akrantz** [comment]
@MsCesarBerard The builds did not complete. Please look into it. Thanks.

**Comment by TCourtneyOwen** [comment]
The reason for the test failures is that the tests timeout with the addition of the test class that builds and installs all projects. I believe the timeout is something like 48 minutes. Given that this timeout occurs too often we need to scale back the scope of the build/install test or see if we can increase the Travis CI timeout.

**Comment by akrantz** [comment]
Is there a way to speed up each test? Is it slow because of npm install running?

Adam

________________________________

**Comment by TCourtneyOwen** [comment]
We could try removing the part that deletes each project folder after it’s built and installed. Deleting the projects takes quite a bit of time. The npm install is required so I don’t see a way to remove that.

On Jul 5, 2018, at 8:48 AM, Adam Krantz <notifications@github.com<mailto:notifications@github.com>> wrote:

Is there a way to speed up each test? Is it slow because of npm install running?

Adam

________________________________

**Comment by TCourtneyOwen** [comment]
@MsCesarBerard Can we get this branch merged into master now? I checked in a fix for test timeouts so you should be good to go.  Once we get this merged, we should publish out a new version of Yo Office

**Comment by MsCesarBerard** [comment]
Yes, I think this is ready to be merged. Is there a way to rerun the tests which timeout with this? just to be sure


---
In reply to: [405293395](https://github.com/OfficeDev/generator-office/pull/335#issuecomment-405293395) [](ancestors = 405293395)

**Comment by akrantz** [comment]
@TCourtneyOwen @MsCesarBerard It looks like this pull request is not picking up the updated Travis CI build definition. I'm going to close this pull request and open a new one.

**Comment by akrantz** [comment]
Reopened as https://github.com/OfficeDev/generator-office/pull/339.

**Comment by TCourtneyOwen** [comment]
Just remove the following confiig from .travis.yml, so we we only run against the stable build of node-js:  6.9.1

**Comment by akrantz** [comment]
OK, the updated master branch just needs to be merged in. I'm going to do that locally to verify and then I'll push to the branch for the new pull request.

**Pull request closed without merging**